Configure

```diff
@@ -743,6 +743,10 @@ while (@argvcopy)
 			{
 			$libs.=$_." ";
 			}
+		elsif (/^-framework$/)
+			{
+			$libs.=$_." ".shift(@argvcopy)." ";
+			}
```

Comment:
Well, -R is also recognized everywhere. In other words, yes, it is kind of "opportunistic", admittedly, but it's not actually more opportunistic that rest of the script :-)